canonical-web-and-design/cookie-policy
# Fix it for other browsers

## Done
Fixed errors on Firefox and Safari

## QA
- Run `gulp build`
- Open the index file in some browsers and check there are no errors in the console

Fixes https://github.com/canonical-webteam/cookie-policy/issues/6
